plugins/calibre.koplugin/wireless.lua

```diff
@@ -707,6 +709,19 @@ function CalibreWireless:sendBook(arg)
     end
 end
 
+function CalibreWireless:sendBookMetadata(arg)
+    local idx = arg.index + 1
+    logger.dbg(string.format("SEND_BOOK_METADATA %d/%d", idx, arg.count), arg)
+
+    if not CalibreMetadata:updateBookIfExists(arg.data) then
+        logger.dbg("Metadata cannot be updated, lpath changed?", arg.data.lpath)
```

Comment:
I don't think this branch is ever executed. IIRC we should prune all entries that are not backed by real files on the device before communicating with the calibre server. So we're sure we always tell the server the correct info.